locales/en.json

```diff
@@ -54,7 +54,12 @@
   },
   "paymentLinksV2": {
     "amount": {
-      "enterAmountToPay": "Enter amount to pay"  
+      "enterAmountToPay": "Enter amount to pay" 
+    },
+    "fieldValidation": {
+      "enterAnAmountInPounds": "Enter an amount in pounds",
```

Comment:
I assume these error messages aren't finalised? It would be confusing to say just "pounds" rather than "pounds and pence" I think, but assume this will change when the error messages are reviewed.

Reply:
Yes - we should review the error messages.
We will also need translations for Welsh.